portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,148 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+<!doctype html>
+<html lang="en">
+
+    <head>
+        <!-- Required meta tags -->
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+        <!-- Bootstrap CSS -->
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.2/css/bootstrap.min.css" integrity="sha384-Smlep5jCw/wG7hdkwQ/Z5nLIefveQRIY9nfy6xoR1uRYBtpZgI6339F5dgvm/e9B"
+            crossorigin="anonymous">
+
+        <link rel="stylesheet" href="style.css">
+
+        <title>Resume</title>
+    </head>
+
+    <body>
+        <main class="container">
+
+            <header>
+
+              <nav>
+                    <ul class="main-nav">
+                        <li class="main-nav__item"><a href="index.html">Home</a></li>
+                        <li class="main-nav__item"><a href="hobby.html">My Hobby</a></li>
+                        <li class="main-nav__item"><a href="test.html">SPS test</a></li>
+                        <li class="main-nav__item"><a href="">Contact me!</a></li>
+                    </ul>
+              </nav>
+
+                <section id="resume-header" class = "row">
+
+                  <figure class="col-3">
+                      <img id="profile-picture" src="./img/foto2x2.png" alt="">
+                  </figure>
+
+                  <article class="col">
+                      <h1>Steven Matias Perez</h1>
+                      <h2>Computer Engineering Student</h2>
+                      <ul>
+                          <li> <b>Mail:</b>  stevenmatias2544@gmail.com</li>
+                          <li> <b>Phone:</b> (787)-462-7560</li>
+                          <li> <b>LinkedIn:</b> <a href=" https://www.linkedin.com/in/steven-mat%C3%ADas-p%C3%A9rez-60b711167/"> <img  class = "social-img"src="./img/linkedinLogo.png"></a></li>
+                          <li> <b>GitHub:</b> <a href="https://github.com/abdiel44"> <img  class = "social-img"src="./img/githubLogo.png"> </a> </li>
+                      </ul>
+                  <article>
+                  
+                </section>
+
+            </header>
```

Comment:
The header looks like it can be reused across multiple pages. You don't have to make any changes now, but if you're interested, there are ways to make a reusable component so that any changes you make only need to be done in one place.